README.pl-PL.md

```diff
@@ -0,0 +1,130 @@
+- [Załączniki](#załączniki)
+  - [Status rozwoju](#status-rozwoju)
+  - [Dodatkowe informacje](#dodatkowe-informacje)
+  - [Przypisy](#przypisy)
+
+<!-- pokaz -->
+<!-- funkcję -->
+<!-- Lista wszystkich komend i funkcji będą dostępne wkrótce. -->
+
+<!-- instalacja & dokumenty -->
+## Budowa z źródła na Linuxie
+### HLShell (główne jądro)
+**Wymagania:**
+ - [Git](https://git-scm.com) system kontroli wersji <!-- nwm czy o to chodziło o co chodziło. "version control system"-->
+ - [Rust toolchain](https://www.rust-lang.org/tools/install)
+
+**Kroki:**
+ 1. Zainstaluj nightly toolchain:
+```bash
+rustup toolchain install nightly
+```
+ 3. Zainstaluj potrzebne komponenty:
+```bash
+rustup component add rust-src --toolchain nightly-x86_64-unknown-linux-gnu && rustup component add llvm-tools-preview --toolchain nightly-x86_64-unknown-linux-gnu && cargo install bootimage
+```
+ 4. Stwórz lokalną kopię repozytorium:
+```bash
+git clone git@github.com:adamperkowski/highlightos.git && cd highlightos
+```
+ 5. Użyj `cd` do folderu `shell/` :
+```bash
+cd shell
+```
+ 7. Zbuduj bootowalny plik binarny :
+```bash
+cargo +nightly bootimage --release
+```
+> Ta komenda stworzy folder `target/target/release` w którym znajdziesz `bootimage-hlshell.bin`, czyli plik binarny.
+
+### HighlightOS ASM
+**Wymagania:**
+ - [Git](https://git-scm.com) system kontroli wersji
+ - [NASM](https://nasm.us) Kompilator Assembly
+
+**Kroki:**
+ 1. Stwórz lokalną kopię repozytorium:
+```bash
+git clone git@github.com:adamperkowski/highlightos.git && cd highlightos
+```
+ 2. Użyj `cd` do folderu `asm/` :
+```bash
+cd asm
+```
+ 3. Skompiluj plik binarny:
+```bash
+nasm -f bin boot.asm -o boot.bin
+```
+> Ta komenda stworzy plik `boot.bin` i umieści go w twoim aktualnym folderze.
+
+## Uruchamianie w QEMU na linuxie
+
+**Wymagania:**
+ - [QEMU](https://www.qemu.org/download/#linux) (cały pakiet)
+ - Bootwalny plik binarny HighlightOS. Możesz takowy pobrać z [wydań](https://github.com/adamperkowski/highlightos/releases), albo [zbudować własny](#Budowa-z-źródła-na-Linuxie).
+
+**Kroki:**
+ 1. Użyj `cd` do folderu z plikiem binarnym.
+ 2. Użyj następującej komendy:
+```bash
+qemu-system-x86_64 -drive format=raw,file=<your_binary_filename>.bin
+```
+> [!IMPORTANT]
+> Upewnij się że zamieniłeś `<nazwa_twojego_pliku_binarnego>` na właściwą nazwę pliku który pobrałeś/zbudowałeś.
+
+## Uruchamianie na prawdziwym sprzęcie
+Możesz również sflaszować plik binarny do swojego USB pedrive'a i uruchomić go na prawdziwej maszynie
+
+Żeby to zrobić wykorzystaj następującą komende:
+```bash
+dd if=<your_binary_filename>.bin of=/dev/sdX && sync
+```
+
+> [!IMPORTANT] <!--**upewnij się...** tak powinno być? -->
+> Upewnij się że zamieniłeś `<nazwa_twojego_pliku_binarnego>` na właściwą nazwę pliku który pobrałeś/zbudowałeś i **upewnij się żeby zamienić `/dev/sdX` na numerem partycji z twojego USB. **Wszelkie dane zostaną starocone!**
```

Comment:
Remove the first `**` here. It's been removed in https://github.com/adamperkowski/highlightos/commit/199ae46df6ebb6ad867eca9115fd72c1f36a81c0.